aten/src/ATen/cuda/detail/CUDAHooks.cpp

```diff
@@ -448,14 +448,18 @@ std::string CUDAHooks::showConfig() const {
 
 
   auto printCudnnStyleVersion = [&](size_t v) {
-    oss << (v / 1000) << '.' << (v / 100 % 10);
+    if (v >= 9000) {
+      oss << (v / 10000) << '.' << (v / 100 % 100);
+    } else {
+      oss << (v / 1000) << '.' << (v / 100 % 10);
+    }
     if (v % 100 != 0) {
       oss << '.' << (v % 100);
     }
   };
```

Comment:
Can we use cudnn_frontend here? No reason to reinvent the wheel: https://github.com/NVIDIA/cudnn-frontend/blob/11b51e9c5ad6cc71cd66cb873e34bc922d97d547/include/cudnn_frontend_shim.h#L444